[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_filter_unsubscribe_from_all_subscribed_content_topics with id: 2024-12-04_11-44-47__cdd78fdd-1e92-4db6-aafc-1f1afc756fce
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.filter:filter.py:28 Running fixture setup: filter_setup
[35mDEBUG   [0m src.steps.filter:filter.py:34 Running fixture setup: setup_main_relay_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-12-04_11-44-47__cdd78fdd-1e92-4db6-aafc-1f1afc756fce__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.104.190
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['22862', '22863', '22864', '22865', '22866']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 22862:22862 -p 22863:22863 -p 22864:22864 -p 22865:22865 -p 22866:22866 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=22864 --rest-port=22862 --tcp-port=22863 --discv5-udp-port=22865 --rest-address=0.0.0.0 --nat=extip:172.18.104.190 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=cded83a1fbbf0df76ebbe5cfbb8fbdb04b5faf4a7ee65decaf47f8bfcf64adad --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=22866 --metrics-logging=true --relay=true --filter=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.104.190 waku ca9923b1059e0e0241e59bf92356e539005632833a22fd33c285abb53f02e1e5
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID ca9923b1059e. Setting up logs at ./log/docker/node1_2024-12-04_11-44-47__cdd78fdd-1e92-4db6-aafc-1f1afc756fce__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 22862
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:22862/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:22862/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.104.190/tcp/22863/p2p/16Uiu2HAm5mU5oB6McVwumLB1aYmFSLsQj4hqo5cvxuJ5x2xSVFHU","/ip4/172.18.104.190/tcp/22864/ws/p2p/16Uiu2HAm5mU5oB6McVwumLB1aYmFSLsQj4hqo5cvxuJ5x2xSVFHU"],"enrUri":"enr:-L24QL8n7j0GTaI2UFMAIm0PVL3ZsoF4OQcUf5XG19n1AfeaFWALXzMYc1TL3DXzq4RAa98h6Iko2ACM4RfpLHtHvcECgmlkgnY0gmlwhKwSaL6KbXVsdGlhZGRyc5YACASsEmi-BllPAAoErBJovgZZUN0DgnJzhQADAQAAiXNlY3AyNTZrMaECmacItq2T4U3eXN1nPxV8sDgqCGaodmXzgCx7UFQNAcODdGNwgllPg3VkcIJZUYV3YWt1MgU"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[35mDEBUG   [0m src.steps.filter:filter.py:39 Running fixture setup: setup_main_filter_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node2_2024-12-04_11-44-47__cdd78fdd-1e92-4db6-aafc-1f1afc756fce__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.18.81
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['23155', '23156', '23157', '23158', '23159']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 23155:23155 -p 23156:23156 -p 23157:23157 -p 23158:23158 -p 23159:23159 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=23157 --rest-port=23155 --tcp-port=23156 --discv5-udp-port=23158 --rest-address=0.0.0.0 --nat=extip:172.18.18.81 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=aca6b142c8ababb1cd3adfba7f47ebffc1e9d16ad8e8cdfcb67e9226b71015b5 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=23159 --metrics-logging=true --relay=false --discv5-bootstrap-node=enr:-L24QL8n7j0GTaI2UFMAIm0PVL3ZsoF4OQcUf5XG19n1AfeaFWALXzMYc1TL3DXzq4RAa98h6Iko2ACM4RfpLHtHvcECgmlkgnY0gmlwhKwSaL6KbXVsdGlhZGRyc5YACASsEmi-BllPAAoErBJovgZZUN0DgnJzhQADAQAAiXNlY3AyNTZrMaECmacItq2T4U3eXN1nPxV8sDgqCGaodmXzgCx7UFQNAcODdGNwgllPg3VkcIJZUYV3YWt1MgU --filternode=/ip4/172.18.104.190/tcp/22863/p2p/16Uiu2HAm5mU5oB6McVwumLB1aYmFSLsQj4hqo5cvxuJ5x2xSVFHU
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.18.81 waku 3c36977b16299d9950cfbfb945f81f5a84523e9d8dd89e6f37344c02a80c648f
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 3c36977b1629. Setting up logs at ./log/docker/node2_2024-12-04_11-44-47__cdd78fdd-1e92-4db6-aafc-1f1afc756fce__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 23155
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:23155/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:23155/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.18.81/tcp/23156/p2p/16Uiu2HAmF7gX8QGzQQ7kCZk1nbSw3aUpvymqvRNaARTxFZXfHB8G","/ip4/172.18.18.81/tcp/23157/ws/p2p/16Uiu2HAmF7gX8QGzQQ7kCZk1nbSw3aUpvymqvRNaARTxFZXfHB8G"],"enrUri":"enr:-L24QEizKn9a_i3atr5xl-DYtBSluY-JcAZ_7_FxJDFAP3klMuKO91Bqlf_wqYnIiLktYJDryXxJycE4KvwqOob58-0CgmlkgnY0gmlwhKwSElGKbXVsdGlhZGRyc5YACASsEhJRBlp0AAoErBISUQZadd0DgnJzhQADAQAAiXNlY3AyNTZrMaEDJI3lE9vpDm7dhq7mTRrjcxtjdfAVs75o01MALNKKrq2DdGNwglp0g3VkcIJadoV3YWt1MgA"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:23155/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.104.190/tcp/22863/p2p/16Uiu2HAm5mU5oB6McVwumLB1aYmFSLsQj4hqo5cvxuJ5x2xSVFHU"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.filter:filter.py:47 Running fixture setup: subscribe_main_nodes
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:22862/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:23155/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "be67e4b3-cc23-4d7b-a1cb-226170e64ea8", "contentFilters": ["/test/1/waku-filter/proto"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"be67e4b3-cc23-4d7b-a1cb-226170e64ea8","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:22862/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:23155/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "713bc31f-c154-4fb9-b88c-fc726ecf0475", "contentFilters": ["Hello World!", "1234567890"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"713bc31f-c154-4fb9-b88c-fc726ecf0475","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:22862/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "1234567890", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:98 Checking that peer NODE_2:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:23155/filter/v2/messages/1234567890" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"RmlsdGVyIHdvcmtzISE=","contentTopic":"1234567890","version":0,"timestamp":1733312704130704896,"ephemeral":false}]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X DELETE "http://127.0.0.1:23155/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "1", "contentFilters": ["Hello World!", "1234567890"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"1","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:22862/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "1234567890", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:98 Checking that peer NODE_2:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:23155/filter/v2/messages/1234567890" -H "Content-Type: application/json" -d 'None'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 400 Client Error: Bad Request for url: http://127.0.0.1:23155/filter/v2/messages/1234567890. Response content: b'Not subscribed to topic: 1234567890'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id ca9923b1059e
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id 3c36977b1629
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container ca9923b1059e. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.